ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/139__trusted_ca_bundles.robot

```diff
@@ -0,0 +1,155 @@
+*** Settings ***
+Documentation    Test Cases to verify Serverless installation
+Library    Collections
+Resource    ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Suite Setup    Suite Setup
+Suite Teardown    Suite Teardown
+
+
+*** Variables ***
+${RHOAI_OPERATOR_NS}    redhat-ods-operator
+${RHOAI_OPERATOR_DEPLOYMENT_NAME}    rhods-operator
+${TEST_NS}    test-trustedcabundle
+${DSCI_NAME}    default-dsci
+${TRUSTED_CA_BUNDLE_CONFIGMAP}    odh-trusted-ca-bundle
+${CUSTOM_CA_BUNDLE}    test-example-custom-ca-bundle
+
+${IS_PRESENT}    0
+${IS_NOT_PRESENT}    1
+
+
+*** Test Cases ***
+Validate Trusted CA Bundles ConfigMaps
```

Comment:
Important: the Test Teardown should restore the initial DSCI bundle configuration found before running this test, even if the test fails in the middle. As an alternative, add the tag `robot:recursive-continue-on-failure` to the test to make sure all keywords are run

Reply:
Refactored setup and teardown for suite and per test case